main/resource-access.bs

```diff
@@ -142,19 +139,151 @@ through POST, using the HTTP `Slug` header as defined in [[!RFC5023]].
 Clients who want the server to determine the URI for their resource, MUST use
 the POST request.
 
-Servers MUST allow creating new resources with a POST request to URI path
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)].
+
+### Reading resources ### {#read}
+
+Servers MUST support the HTTP `GET`, `HEAD` and `OPTIONS` methods [[!RFC7231]]
+for clients to read resources or to determine communication options.
+[[Source](https://github.com/solid/specification/issues/39#issuecomment-538017667)]
+
+When using Web Access Control ([[#wac]]):
+
+Servers MUST indicate their support for HTTP Methods by responding to HTTP
+`GET`, `HEAD` and `OPTIONS` requests on the resource's URL with the HTTP
+Method tokens in the HTTP response header `Allow`.
```

Comment:
On what methods?

Reply:
Originally this sentence was in the same paragraph as the former. I moved it to make it more clear for review. But I agree that it is best to specify if it stays in its own paragraph. If I moved it to the previous paragraph, would that address your question?

Reply:
Yes, it would.

Reply:
(Especially the "When using Web Access Control ([[#wac]]):" interjection was difficult in that sense.)

Reply:
https://github.com/solid/specification/pull/188/commits/5ec6ad6536d5fbb4361399f432c7b7d67a6ec665